tests/test_adjointA.py

```diff
@@ -29,4 +29,4 @@ def test_acoustic(dimensions, time_order, space_order, fix_dim):
     term1 = np.dot(srca.data.reshape(-1), solver.source.data)
     term2 = linalg.norm(rec.data) ** 2
     print(term1, term2, ("%12.12f") % (term1 - term2), term1 / term2)
-    assert np.isclose(term1 / term2, 1.0, atol=0.001)
```

Comment:
OK, so, I double checked and you're right, it turns out this change was actually unnecessary so I removed it.

The reason it was there was briefly discussed on Slack, when I was talking about parallelising the sources loop (not the receiver loops). 

So this is not a problem any more